SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorSettings.cpp

```diff
@@ -38,12 +38,34 @@ void FSpatialLaunchConfigDescription::OnWorkerTypesChanged()
 	}
 }
 
+FRuntimeVariantVersion::FRuntimeVariantVersion() = default;
+
+const FString& FRuntimeVariantVersion::GetVersionForLocal() const
+{
+	if (bUseGDKPinnedRuntimeVersion || LocalRuntimeVersion.IsEmpty())
+	{
+		return PinnedVersion;
+	}
+	return LocalRuntimeVersion;
+}
+
+const FString& FRuntimeVariantVersion::GetVersionForCloud() const
+{
+	if (bUseGDKPinnedRuntimeVersion || CloudRuntimeVersion.IsEmpty())
+	{
+		return PinnedVersion;
+	}
+	return CloudRuntimeVersion;
+}
+
 USpatialGDKEditorSettings::USpatialGDKEditorSettings(const FObjectInitializer& ObjectInitializer)
 	: Super(ObjectInitializer)
 	, bShowSpatialServiceButton(false)
 	, bDeleteDynamicEntities(true)
 	, bGenerateDefaultLaunchConfig(true)
-	, bUseGDKPinnedRuntimeVersion(true)
+	, RuntimeVariant(ESpatialOSRuntimeVariant::Distributed)
+	, DistributedRuntimeVersion(SpatialGDKServicesConstants::SpatialOSRuntimePinnedVersion)
+	, SingleNodeRuntimeVersion(SpatialGDKServicesConstants::SpatialOSRuntimePinnedVersion)
```

Comment:
This constant changes when merging with Andreas's PR yeah?

Reply:
Yes.

Reply:
It's also been noted that Squid and Kraken will require different template names which we should update the defaults to when swithcing